### DISCUSSION TITLE
How to remove .html extension

### DISCUSSION THREAD

**Discussion by iqlal** [vuejs/vitepress#2456] [Q&A]
How to remove .html extension in my all pages

![image](https://github.com/vuejs/vitepress/assets/64581779/91cb3887-8f53-4d70-a5e8-6155a71f205d)

**Comment by zonemeen** [comment]
You can configure it like this:

```ts
export default defineConfig({
  cleanUrls: true
})
```

Related Link: https://vitepress.dev/reference/site-config#cleanurls

**Reply by iqlal** [reply]
where I configure that?

Ps. I don't have a webdev background.🙏

**Reply by peiyanlu** [reply]
Via the path .vitepress/config.ts

**Reply by iqlal** [reply]
Ok Thanks
